### ISSUE TITLE
have a way to set options to default.. 

### ISSUE THREAD

**Issue by aaronsta1** [ZoneMinder/zoneminder#2290]
so i changed my default storage folder and in doing so i deleted the actual default folder.. 

now that its deleted, i cant access any of the video that was on there, default is no longer in the drop down menu, and while its not an issue i cant find a way to delete the space they are taking up.

the default storage is still shown on the top bar.. and it says 20GB is being taken up in events.

**Comment by SteveGilvarry** [comment]
Default is not truly a storage area as I see it. I also removed mine, but the GUI still reports on that disk usage, which happens to be the same disk as /. So when I see 61% that is true. But there is nothing in the folder itself, when you say you deleted the actual default folder do you mean the storage area listing for default in options or the actual folder on disk? I did the first one.

**Comment by aaronsta1** [comment]
yeah in the options.. but when i hover over the default in the GUI it says events is taking up 20GB of space. 
i have no idea where the actual folder is on the disk.. i sort of figured when i deleted the default space it would delete the files off the drive..

**Comment by SteveGilvarry** [comment]
I know I saw discussion in Slack about changing from du to using DB based event size stuff, not sure if it is in a release yet, or was in the console header. Jump on zm slack and ask there if you want faster answer.
Ubuntu default is /var/cache/zoneminder/events, my du shows nothing, but I see the same df (59% vs 61% in ZM) not matching. I will assign to myself and investigate it, since I get the same thing and just haven't bothered to look at why I still see default.

**Comment by SteveGilvarry** [comment]
Triaging the issue backlog. This describes the 1.32-era storage UI; disk-usage reporting moved to DB-based event sizes and the described behaviour no longer exists. Closing.
